accounting/accounting.go

```diff
@@ -262,12 +278,14 @@ func (accounting *AccountingState) AddTransaction(txnr *idb.TxnRow) (err error)
 			accounting.updateAlgo(stxn.Txn.CloseRemainderTo, int64(stxn.ClosingAmount))
 		}
 		if stxn.ReceiverRewards != 0 {
-			accounting.updateAlgo(stxn.Txn.Receiver, int64(stxn.ReceiverRewards))
-			accounting.updateAlgo(accounting.rewardAddr, -int64(stxn.ReceiverRewards))
+			v := int64(stxn.ReceiverRewards)
+			accounting.updateAlgoAndRewards(stxn.Txn.Receiver, v, v)
+			accounting.updateAlgo(accounting.rewardAddr, -v)
 		}
 		if stxn.CloseRewards != 0 {
-			accounting.updateAlgo(stxn.Txn.CloseRemainderTo, int64(stxn.CloseRewards))
-			accounting.updateAlgo(accounting.rewardAddr, -int64(stxn.CloseRewards))
+			v := int64(stxn.CloseRewards)
+			accounting.updateAlgoAndRewards(stxn.Txn.CloseRemainderTo, v, v)
+			accounting.updateAlgo(accounting.rewardAddr, -v)
```

Comment:
Since it's repeated multiple times, consider making it a function.

Reply:
Done